templates/s3/lambda_access_bucket_policy.json.tpl

```diff
@@ -8,7 +8,8 @@
       },
       "Action": [
         "s3:GetObject",
-        "s3:PutObject"
+        "s3:PutObject",
+        "s3:ListBucket"
       ],
       "Resource": "arn:aws:s3:::${bucket_name}/*"
```

Comment:
You need to add a resource for the bucket as well as the objects. `"arn:aws:s3:::${bucket_name}"`